src/pages/lesson-plans/step.html

```diff
@@ -0,0 +1,10 @@
+<div
+  class="step-content"
+  [ngClass]="{
+    subdued:  (currentQuestionStepId === null && done) || (currentQuestionStepId !== null && currentQuestionStepId !== stepId),
+    highlighted: currentQuestionStepId === stepId || (currentQuestionStepId === null && highlightedStepId === stepId)
+  }"
+  (click)="clickStep()"
+>
+  <ng-content></ng-content>
```

Comment:
draft of the template for the Step component -- if you haven't run into `ng-content` yet, it can be used as above to include the contents of the component as created by the parent class.  It can also be used for more powerful things, but aren't needed here.  (This will need to be used for the LessonPlan, at least as I've set things up.)

This is what I used to figure it out (though I knew what I was looking for because I'm familiar with a similar feature from angular 1): https://medium.com/claritydesignsystem/ng-content-the-hidden-docs-96a29d70d11b